sm2a/infrastructure/main.tf

```diff
@@ -19,7 +19,7 @@ resource "random_password" "password" {
 
 
 module "sma-base" {
-  source                         =  "https://github.com/NASA-IMPACT/self-managed-apache-airflow/releases/download/v1.1.3/self-managed-apache-airflow.zip"
+  source                         =  "https://github.com/NASA-IMPACT/self-managed-apache-airflow/releases/download/v1.1.4/self-managed-apache-airflow.zip"
```

Comment:
Upgrade SM2A base module to force redeploy the services (avoid docker image hash 404 error)